src/emulator/functionsEmulatorShared.ts

```diff
@@ -49,7 +49,7 @@ export interface FunctionsRuntimeArgs {
 export interface FunctionsRuntimeBundle {
   projectId: string;
   proto?: any;
-  triggerId?: string;
+  triggerId: string;
```

Comment:
Diagnostic workers have traditionally had `undefined` triggerId.  I would love to see this change but can you explain a bit?  This is a big change for a logging PR.

Reply:
I remembered that as well and I went through and tested this change, there's no where in our code that I can find that requires triggerId to be nullable, can you link me to where we expect this to be null?